ADRs/Meal inventory updates.md

```diff
@@ -0,0 +1,66 @@
+# Meal Inventory Updates
+
+## Status
+
+_PROPOSED_
+
+## Context and Problem Statement
+
+Farmacy Foods produces meals in one or more central kitchens, delivers those meals to various locations, where they are sold to customers via a smart fridge or point-of-sale system.
+The meals are the central aspect of the business, and there is a big focus on food quality.
+
+### Requirements
+
+* Each meal will be tracked from the point of its creation in the kitchen, throughout delivery, stocking the fridge, and purchase by the customer.
+* Customer can give feedback on a meal after buying it.
+* Customers can search for available meals nearby.
+* Some history of the meal should be recorded, such as where it was produced, for tracking purposes.
+
+### Business Assumptions
+
+* Inventory changes are not too frequent, since they are tied to movement of a physical meal - like an employee stocking the fridge, or a customer buying a meal.
+* Meals are generally produced and purchased in a single area. There is no interaction between the databases.
```

Comment:
No interaction between which databases? Unclear wording.

Reply:
That's about the different DBs for each metro area. I'll remove it.